runtime/lua/vim/_defaults.lua

```diff
@@ -298,7 +298,11 @@ if tty then
       end,
     })
 
-    io.stdout:write('\027]11;?\007')
+    local query = '\027]11;?\007'
+    if os.getenv('TMUX') then
```

Comment:
nitpicking: when `TMUX=""` (not `nil`), this will also evaluate as truthy. Consider:

```diff
-os.getenv('TMUX')
+#(os.getenv('TMUX') or "") > 0
``

Reply:
The convention is that `$TMUX` is only ever set when a tmux session is active. If a user has `$TMUX` set to an empty string outside of a tmux session then they did so manually (or it is the result of a buggy configuration). In either case, it's not something I think we need to handle here.